xml/book_sles_containers.xml

```diff
@@ -22,6 +22,7 @@
   <xi:include href="common_copyright_gfdl.xml"/>
   <abstract>
    <para>
+    This guide provides an introduction to the &suse; container ecosystem.
```

Comment:
I do not like this word -- it sounds too much like marketing to me. However, any replacement I can think of would be a whole sentence.

Reply:
I just moved over to the abstract what was already written elsewhere (before the sentence was in container_intro.xml). I think we have another chance to fix it when the work on the guide continues.